src/index.js

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
 import { render } from 'react-dom';
+import App from './components/App';
```

Comment:
Oops, the global block of imports should be separated from the local imports by an empty line